src/nvim/highlight_group.c

```diff
@@ -222,6 +222,7 @@ static const char *highlight_init_both[] = {
   "default link LspReferenceWrite           LspReferenceText",
   "default link LspReferenceTarget          LspReferenceText",
   "default link LspSignatureActiveParameter Visual",
+  "default link LspLinkedEditingRange       IncSearch",
```

Comment:
seems over-specific. are we certain we need this highlight or can we skip it for now (just use IncSearch)